mkosi/__init__.py

```diff
@@ -411,25 +411,39 @@ def fedora_release_cmp(a: str, b: str) -> int:
 
 # Debian calls their architectures differently, so when calling debootstrap we
 # will have to map to their names
+# uname -m -> dpkg --print-architecture
 DEBIAN_ARCHITECTURES = {
-    "x86_64": "amd64",
-    "x86": "i386",
     "aarch64": "arm64",
     "armhfp": "armhf",
+    "armv7l": "armhf",
+    "ia64": "ia64",
+    "mips64": "mipsel",
+    "m68k": "m68k",
```

Comment:
I only had a quick look, but m68k is not in den DPS, right? If it's not in there (yet?), maybe we should restrict ourselves to architectures that are.

Reply:
we can add it - but there's not that many architectures, and they rarely change, so I'd rather just have them all and be done with it, since it's just two tables so doesn't take that much effort